dll/win32/dnsapi/CMakeLists.txt

```diff
@@ -2,16 +2,16 @@
 include_directories(
     include
     ${REACTOS_SOURCE_DIR}/sdk/include/reactos/idl
-    ${REACTOS_SOURCE_DIR}/sdk/lib/3rdparty/adns/src
-    ${REACTOS_SOURCE_DIR}/sdk/lib/3rdparty/adns/adns_win32)
```

Comment:
You should then remove those corresponding files as well from the source tree? Or are they also used elsewhere in our code?